### PR TITLE
Update information-barriers-policies.md

### PR DESCRIPTION
**Pull request by DPascua-MSFT**
Example starting at line 419 creates a policy mismatch error.
Either 
Manufacturing can talk to everyone and only Sales and Research block each other 
OR 
Manufacturing can only talk to HR and Marketing, blocking Sales and Research.
Have proposed here how to make a blend of both as this is a frequently requested scenario within Support.

Three additional changes proposed:
-Expanding "Everyone" in HR and Marketing and "Anyone other than HR or Marketing" in Manufacturing would help clear up this understanding and reflect how a policy is made in UI or PowerShell.
-Using a naming convention like "Sales_Policy" to match the assigned segment allows for changes in the policy, such as including Manufacturing to correct the example, to make sense since policy names cannot change without first deleting the entire policy and recreating from scratch.
-Identify why HR and Marketing do not need policies.

### CODE REVIEW

**Comment by ghost** [comment]
[CLA assistant check](https://cla.opensource.microsoft.com/MicrosoftDocs/microsoft-365-docs?pullRequest=8619) <br/>All CLA requirements met.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/information-barriers-policies.md [hunk 1]
```diff
@@ -418,19 +418,19 @@ Contoso has five departments: *HR*, *Sales*, *Marketing*, *Research*, and *Manuf
...
 
-1. An IB policy designed to prevent Sales from communicating with Research
-2. Another IB policy to prevent Research from communicating with Sales.
-3. An IB policy designed to allow Manufacturing to communicate with HR and Marketing only.
+1. An IB policy designed to prevent Sales from communicating with Research.
+2. An IB policy to prevent Research from communicating with Sales and Manufacturing.
+3. An IB policy designed to allow Manufacturing to communicate with HR, Sales, and Marketing only.
 
-For this scenario, it's not necessary to define IB policies for *HR* or *Marketing*.
+For this scenario, it's not necessary to define IB policies for *HR* or *Marketing* as IB policies without a defined policy has an implied allow all communication with all other segments.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md:433]:
```suggestion
For this scenario, it's not necessary to define IB policies for *HR* or *Marketing* as IB policies without a defined policy have an implied allow all communication with all other segments.
```

File: microsoft-365/compliance/information-barriers-policies.md [hunk 2]
```diff
@@ -452,9 +452,9 @@ Contoso defines three IB policies, as described in the following table:
...
+| **Policy 2: Prevent Research from communicating with Sales and Manufacturing** | `New-InformationBarrierPolicy -Name "Research_Policy" -AssignedSegment "Research" -SegmentsBlocked "Sales","Manufacturing" -State Inactive` <p> In this example, the IB policy is called *Research_Policy*. When this policy is active and applied, it will help prevent users who are in the Research segment from communicating with users in the Sales or Manufacturing segments. This is a one-way policy with Manufacturing; this policy won't prevent Manufacturing from communicating with Research as Policy 3 is needed. |
+| **Policy 3: Allow Manufacturing to communicate with HR and Marketing only** | `New-InformationBarrierPolicy -Name "Manufacturing_Policy" -AssignedSegment "Manufacturing" -SegmentsAllowed "HR","Sales","Marketing","Manufacturing" -State Inactive` <p> In this case, the IB policy is called *Manufacturing_Policy*. When this policy is active and applied, Manufacturing can communicate only with HR, Sales, and Marketing. Allow and Block policies can be mixed in this way as long as symmetry is met, as outlined in the table above. |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md]:
```suggestion
| **Policy 3: Allow Manufacturing to communicate with HR and Marketing only** | `New-InformationBarrierPolicy -Name "Manufacturing_Policy" -AssignedSegment "Manufacturing" -SegmentsAllowed "HR","Sales","Marketing","Manufacturing" -State Inactive` <p> In this case, the IB policy is called *Manufacturing_Policy*. When this policy is active and applied, Manufacturing can communicate only with HR, Sales, and Marketing. Allow and Block policies can be mixed in this way as long as symmetry is met, as outlined in the previous table. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DamienPascua, kindly commit the minor suggested changes. Many thanks.

File: microsoft-365/compliance/information-barriers-policies.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md]:
```suggestion
For this scenario, it's not necessary to define IB policies for *HR* or *Marketing* as IB policies without a defined policy have an implied allow all communication with all other segments.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@DamienPascua, kindly commit the remaining suggested change. Thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by DPascua-MSFT** [comment]
@JohanFreelancer9 Looking to confirm we are good on this pull request, do I need to take any additional action?

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by robmazz** [comment]
@yogkumgit, @DamienPascua - thanks for the feedback, will need to confirm these suggestions with PM and DEV.

**Comment by robmazz** [comment]
@DamienPascua - thanks for the feedback. This topic is undergoing changes for the newer version of IB, I will discuss this the PG and confirm that this addition is still accurate. Thanks again.

**Pull request closed without merging**